### ISSUE TITLE
Add support for `junitparser merge` to print to `stdout`

### ISSUE THREAD

**Issue by augfab** [weiwei/junitparser#62]
it would be nice for use of `junitparser merge` in shell scripts (_e.g._ using pipes) if it could print to `stdout`, for example by means of sing the pseudo-filename `-`, a common Unix practice. This would remove the need to use a temporary intermediate file. I think this could done with [`argparse.FileType`](https://docs.python.org/3/library/argparse.html#filetype-objects).

`a.xml`
```xml
<?xml version='1.0' encoding='utf-8'?>
<testsuite errors="0" failures="0" name="JUnit" skipped="0" tests="1" time="0.0">
  <testcase classname="a" name="a()" time="0.0" />
</testsuite>
```
`b.xml`
```xml
<?xml version='1.0' encoding='utf-8'?>
<testsuite errors="0" failures="0" name="JUnit" skipped="0" tests="1" time="0.0">
  <testcase classname="b" name="b()" time="0.0" />
</testsuite>
```

What I expect:
```
$ junitparser merge a.xml b.xml -
<?xml version='1.0' encoding='utf-8'?>
<testsuites errors="0" failures="0" skipped="0" tests="2" time="0.0"><testsuite errors="0" failures="0" name="JUnit" skipped="0" tests="2" time="0.0"><testcase classname="a" name="a()" time="0.0" />
<testcase classname="b" name="b()" time="0.0" />
$ cat ./-
cat: ./-: No such file or directory
```

What happens:
```
$ junitparser merge a.xml b.xml -
$ cat ./-
<?xml version='1.0' encoding='utf-8'?>
<testsuites errors="0" failures="0" skipped="0" tests="2" time="0.0"><testsuite errors="0" failures="0" name="JUnit" skipped="0" tests="2" time="0.0"><testcase classname="a" name="a()" time="0.0" />
<testcase classname="b" name="b()" time="0.0" />
</testsuite></testsuites>
```

Final use case I have in mind (format merged XML with [`xmllint(1)`](http://xmlsoft.org/xmllint.html)):
```bash
junitparser merge a.xml b.xml - | xmllint --format > junit.xml
```

**Comment by augfab** [comment]
I'm using version 2.0.0.
```
$ junitparser --version
junitparser 2.0.0
```

**Comment by weiwei** [comment]
@augfab Implemented in 2.1.0. Please check it out.

Cheers.

**Comment by augfab** [comment]
I confirm it works, thanks a lot @weiwei :tada:.
